compiler/sizealignoffsetimpl.nim

```diff
@@ -408,8 +408,25 @@ proc computeSizeAlign(conf: ConfigRef; typ: PType) =
       typ.size = typ.lastSon.size
       typ.align = typ.lastSon.align
       typ.paddingAtEnd = typ.lastSon.paddingAtEnd
-
-  of tyGenericInst, tyDistinct, tyGenericBody, tyAlias, tySink, tyOwned:
+  
+  of tyGenericInst:
+    var index = typ.len - 1
+    if sfSizeOf in typ[0].sym.flags:
+      var id:int
+      for n in typ[0].sym.ast[0][1].sons:
+        if n.kind == nkCall and n[0].kind == nkIdent and n[0].ident.s == "sizeOf":
```

Comment:
This works with `sizeof()` ?. 🤔

Reply:
I mean if code uses `sizeof()` instead of `sizeOf()`.

Reply:
hmm, didn't aware of that,  `system.sizeof` is all lower case pragmas all camel case. so it's no.

Reply:
Should it ?. 🤔 Nim is case insensitive.

Reply:
dunno the answer, there're lot of places in compiler using string comparison